### PR TITLE
US 1583733 Add missing language IDs - 41

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "console" for program output displayed on the console. 
- Used "console" for command-line tool examples.

Contributes to #2192

### CODE REVIEW

File: docs/fsharp/style-guide/conventions.md [hunk 1]
```diff
@@ -352,7 +352,7 @@ In contrast to the previous point, partial application is a wonderful tool for r
 
 For example, consider the following solution topography:
 
-```
+```console
```
Comment (mairaw) [inline docs/fsharp/style-guide/conventions.md]: I wouldn't add an identifier here

File: docs/fsharp/tutorials/type-providers/creating-a-type-provider.md [hunk 2]
```diff
@@ -894,7 +894,7 @@ let function1 () =
 
 Here’s an image of the resulting code decompiled by using ildasm.exe:
 
-```
+```console
```
Comment (mairaw) [inline docs/fsharp/tutorials/type-providers/creating-a-type-provider.md]: I think you've applied IL before to similar cases, no?

```suggestion
```il
```

File: docs/fsharp/tutorials/type-providers/creating-a-type-provider.md [hunk 3]
```diff
@@ -928,13 +928,13 @@ Observe the following conventions when authoring type providers.
 
 **Providers for Connectivity Protocols** In general, names of most provider DLLs for data and service connectivity protocols, such as OData or SQL connections, should end in `TypeProvider` or `TypeProviders`. For example, use a DLL name that resembles the following string:
 
-```
+```fsharp
```
Comment (mairaw) [inline docs/fsharp/tutorials/type-providers/creating-a-type-provider.md]: I wouldn't add an identifier here. Perhaps use backticks for this one?

File: docs/fsharp/tutorials/type-providers/creating-a-type-provider.md [hunk 4]
```diff
@@ -1123,7 +1123,7 @@ You can invoke type providers by using the following tools:
 
 You can often debug type providers most easily by using fsc.exe on a test script file (for example, script.fsx). You can launch a debugger from a command prompt.
 
-```
+```console
   devenv /debugexe fsc.exe script.fsx
```
Comment (mairaw) [inline docs/fsharp/tutorials/type-providers/creating-a-type-provider.md]:
```suggestion
devenv /debugexe fsc.exe script.fsx
```

**Review by mairaw — COMMENTED** [review]
Leaving some comments but would prefer to have @cartermp to double check these changes too. Thanks!

**Comment by v-thpra** [comment]
@mairaw I made all of your suggested fixes. Thanks!

**Review by rpetrusha — APPROVED** [review]
LGTM, @v-thpra. I'll merge your PR now.

**Pull request merged**